app/Controllers/InfoController.js

```diff
@@ -5,6 +5,7 @@ const sendInfo = catchAsync(async (_req, res) => {
   res.send({
     identifier: 'vocascan-server',
     version: getVersion(),
+    locked: process.env.LOCKED === 'true',
```

Comment:
you need to change that to the new variable name. (look also for other places). And we also need to make sure that the variable is always parsed the same way. 
`Boolean("false")` is not the same as `"false" === "true"`. So for now I think `=== "true"` would be the best solution. But later we can add a util function to parse boolean envs to return also false for 0 and true for 1.